src/main/java/com/lightstep/opentelemetry/exporter/LightStepSpanExporter.java

```diff
@@ -388,9 +412,28 @@ private URL getCollectorUrl() throws MalformedURLException {
      */
     public LightStepSpanExporter build() throws MalformedURLException {
       defaultDeadlineMillis();
-      defaultComponentName();
+      setDefaultComponentName();
       return new LightStepSpanExporter(
           getCollectorUrl(), deadlineMillis, accessToken, okHttpDns, componentName);
     }
+
+    /**
+     * Installs exporter into tracer SDK provider with batching span processor.
+     *
+     * @param tracerSdkProvider tracer SDK provider
+     */
+    public void install(TracerSdkProvider tracerSdkProvider) throws MalformedURLException {
```

Comment:
Could we provide an overload that takes no parameters, and installs the default one? i.e. `OpenTelemetry.getProvider()` (which users will be using the majority of the time)

Reply:
done